### PR TITLE
feat: worked on member improvements

### PR DESCRIPTION
**Pull request by vvaibhavdesai**
- [x]  made navbar sticky
- [x] accordions open up based on the default behaviour 
- [ ] icons for members are added linking for those needs to be done

image

### CODE REVIEW

File: src/components/MemberProfileCard/index.tsx [hunk 1]
```diff
@@ -1,22 +1,30 @@
...
+import GITHUB_ICON from "./assets/github_icon.svg";
+
+import styles from "./memberprofile.module.css";
 
 /**
  *
  * @returns a JSX component of member detail on members profile page
  */
 export default function index({ userData }: { userData: any }) {
+  console.log(userData,"userData")
```
Comment (anishpawaskar) [inline src/components/MemberProfileCard/index.tsx]: remove console

File: src/components/Layout/Navbar/NavbarDesktop.tsx [hunk 2]
```diff
@@ -42,34 +40,36 @@ export default function NavbarDesktop() {
   ));
 
   return (
-    <UnorderedList
-      listStyleType='none'
+   <nav style={{position:'fixed',width:'100%', zIndex:'1'}}>
+     <UnorderedList
+      listStyleType="none"
       sx={{
```
Comment (anishpawaskar) [inline src/components/Layout/Navbar/NavbarDesktop.tsx:46]: Please can you put this into module css

**Review by anishpawaskar — CHANGES_REQUESTED** [review]

File: src/components/MemberContribution/index.tsx [hunk 3]
```diff
@@ -1,32 +1,41 @@
...
+       <ContributionAccordion
+        accordionTitle={'Active tasks'}
+        contribution={userActiveTask}
+       fallBackLabel={"No active contributions found 😢"}
       />
-
       <ContributionAccordion
         accordionTitle={'All Contribution'}
         contribution={all}
+        fallBackLabel={"How is this person a member? 🤔"}
```
Comment (vinayak-trivedi) [inline src/components/MemberContribution/index.tsx:38]: let's add these fallback labels in constants

**Review by anishpawaskar — APPROVED** [review]
LGTM

**Review by vinayak-trivedi — APPROVED** [review]
LGTM

**Pull request merged**